kvbc/include/pruning_handler.hpp

```diff
@@ -75,7 +72,7 @@ class RSAPruningVerifier {
  private:
   struct Replica {
     std::uint64_t principal_id{0};
-    bftEngine::impl::RSAVerifier verifier;
+    std::unique_ptr<concord::util::crypto::IVerifier> verifier;
```

Comment:
If a class is `RSAPrunningVerifier`, shouldn't there be a pointer to `concord::util::crypto::RSAVerifier`?

Reply:
We don't.
I didn't want to have too much changes, so I left it to a future refactoring